### PR TITLE
Rate limit enqueue subjects worker

### PR DESCRIPTION
**Pull request by camallen**
specifically the non-logged in queue, allow only 1 every 10 seconds by default. i have built and setup the config file for this.

### CODE REVIEW

**Comment by camallen** [comment]
@marten can you take a quick look?

File: config/initializers/subject_enqueue_config.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      @config ||= begin
+                    file = Rails.root.join('config/subject_enqueue_config.yml')
+                    YAML.load(File.read(file))[Rails.env].symbolize_keys
+                  rescue Errno::ENOENT, NoMethodError
+                    {  }
+                  end
+    end
+
+    def self.congestion_opts
+      config[:congestion_opts].symbolize_keys
```
Comment (marten) [inline config/initializers/subject_enqueue_config.rb]: Rescuing and defaulting to an empty hash in `config` won't help much if you end up calling symbolize_keys on nil here.

Comment (camallen) [inline config/initializers/subject_enqueue_config.rb]: good point, will fix

**Comment by marten** [comment]
LGTM otherwise

**Pull request merged**